### PR TITLE
Feature/final program

### PR DESCRIPTION
**Pull request by karolkozak**
Inegrated two files: camera.py and serverToMobile.py. Everything works excellent, so it could be easily merge!

### CODE REVIEW

File: Full Program/finalFile.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+print("Photo mode\n")
+
+# BLUETOOTH SETUP
+server_sock = BluetoothSocket(RFCOMM)
+server_sock.bind(("", PORT_ANY))
+server_sock.listen(1)
+
+port = server_sock.getsockname()[1]
+
+uuid = "94f39d29-7d6d-437d-973b-fba39e49d4ee"
```
Comment (MarcinLason) [inline Full Program/finalFile.py:100]: I think that this value should be more visible (maybe defined on the top of file?) and in final application we have to allow user to change that value in another way than editing the code :)

File: Full Program/finalFile.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                capturePicture()
+            elif MODE_FLAG == 1:   # taking video
+                captureVideo()
+            elif MODE_FLAG == 2:   # taking video after motion detected
+                captureWithPIR()
+        elif data == 'turnOff':
+            data = 'turn off!'
+            cleanUp()
+        else:
+            data = 'WTF!'
```
Comment (MarcinLason) [inline Full Program/finalFile.py:140]: WTF!? Seriously? 💃

**Review by MarcinLason — APPROVED** [review]
finalFile is probably not a best filename, but approve ;)

File: Full Program/finalFile.py (hunk 1 above)
Comment (karolkozak) [reply Full Program/finalFile.py:100]: I cannot agree with the last one. UUID must not be changeable, 'cause this number must be identical as in mobile app and there UUID is hardcoded (it should be).
Only variable that can be accessible is name of Raspi's Bluetooth in mobile app.

File: Full Program/finalFile.py (hunk 2 above)
Comment (karolkozak) [reply Full Program/finalFile.py:140]: Seriously! Let's get fun!

P. S. I'll change it ;)

**Pull request merged**